simulator/docs/environment-variables.md

```diff
@@ -5,6 +5,10 @@ Until deprecation, the simulator will read the configuration in the environment
 if the environment variable is not set, it will read the configuration in the configuration file.
 For config file, please refer to the simulator [config.yaml](./../config.yaml).
 
+> [!WARNING]
+> `KUBE_API_HOST` and `KUBE_API_PORT` are deprecated. They will be removed in the near future.
+> Please use `KUBE_APISEVER_URL` instead.
```

Comment:
```suggestion
> To reduce the maintenance burden, the simulator no longer run up kube-apiserver internally 
> and we made the simulator require the kube-apiserver outside.
> `KUBE_API_HOST` and `KUBE_API_PORT` are deprecated. They will be removed in the near future, meanwhile they're just ignored.
```

Reply:
> meanwhile they're just ignored.

I have a simple question. Do you plan to ignore `KUBE_API_HOST` and `KUBE_API_PORT` until both values is removed ?

https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/333/files#diff-1cb29cb30fedf9a3629f7ebbaca8a15317fa8fa4a03b2aa07b81ba3438d42759L139-R144

The current changes prioritize `KUBE_APISERVER_URL` if the values is set. In other hand, if the values isn't set, we use `KUBE_API_HOST` and `KUBE_API_PORT` to keep backward compatibility. So quoted sentence may confuse to someone that use kube-scheduler-simulator.  Or do I should change the implementation not to use both configuration values ? What do you think ?

Reply:
> if the values isn't set, we use KUBE_API_HOST and KUBE_API_PORT to keep backward compatibility.

It changes the meaning of these configurations, which is not great. 
The previous meaning was they're the values to configure the kube-apiserver which the simulator internally starts and we don't have such internal kube-apiserver anymore. So, I'd say just ignoring them is the simplest idea here.

Reply:
OK, I understand what you said. 
It's exactly right, so I'll modify this explanation.

Reply:
Thanks @sanposhiho